src/Footing.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+function Footing(props) {
+  return (
+    <footer>
+      <ul>
+        {props.address.map(adres => (
+          <li>{adres}</li>
+        ))}
```

Comment:
There's something missing here, and you might see an error in the JS console. Can could you fix it?

Reply:
Thanks, Ali, I think It has been solved now.